setup.py

```diff
@@ -88,5 +88,6 @@ def hello():
     entry_points='''
         [console_scripts]
         flask=flask.cli:main
+        ext_migrate=scripts.flaskext_migrate:fix
```

Comment:
command name is to general, also extras should be used and the command should be marked with a extra